dts/bindings/led/dac-leds.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2018, Linaro Limited
+# Copyright (c) 2025, Prevas A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Group of DAC-controlled LEDs.
+
+  Each LED is defined in a child node of the dac-leds node.
+
+  Here is an example which defines an LED in the node /leds:
+
+  / {
+          leds {
+                  compatible = "dac-leds";
+                  led_0 {
+                          dac = <&dac1>;
+                          channel = <0>;
+                          resolution = <12>;
+                          max-voltage-mv = <3300>;
+                          forward-voltage-mv = <2100>;
+                          resistor-forward-voltage-mv = <(5 * 120)>;
+                  };
+          };
+  };
+
+  Above:
+
+  - led_0 uses dac1 channel 0 with 12 bit resolution.
+
+compatible: "dac-leds"
+
+include: ["dac-leds-common.yaml"]
+
+child-binding:
+  description: DAC LED child node
+  properties:
+    max-voltage-mv:
+      type: int
+      required: true
+      description: |
+        Maximum voltage in millivolt (the DAC's maximum output voltage).
+    forward-voltage-mv:
+      type: int
+      required: true
+      description: |
+        LED forward voltage in millivolt.
```

Comment:
An LED's forward voltage is not constant, but varies with current, as shown in the images posted by @bjarki-andreasen. If we decide to specify the LED like this, we'll at least need a more clear description.